pages/index.js

```diff
@@ -16,28 +16,34 @@ export default function HomePage() {
 
   return (
     <>
-      <Map />
-      <AddItemLink href="/create">
-        <StyledLinkTitle>Add item</StyledLinkTitle>
-      </AddItemLink>
-      <StyledList>
-        {isLoading ? (
-          <li>
-            <h2>Loading...</h2>
-          </li>
-        ) : (
-          items
-            .slice() // temporary reverse function!
-            .reverse()
-            .map((item) => (
-              <li key={item.itemId}>
-                <ItemLink href={`/items/${item.itemId}`}>
-                  <Item title={item.title} initialStatus={item.initiallyLost} />
-                </ItemLink>
-              </li>
-            ))
-        )}
-      </StyledList>
+      {isLoading ? (
+        <li>
+          <h2>Loading...</h2>
+        </li>
+      ) : !listView ? (
+        <Map items={items} listView={listView} />
+      ) : (
+        <>
```

Comment:
In general I would recommend not chaining ternary operators, since it is hard to read. You can do something like this instead:
```
if (isLoading) {
   return <h2>Loading...</h2>
}

return ...
```